templates/signup.html

```diff
@@ -104,6 +104,45 @@ <h4 class="text-blue-500 mt-0 mb-0 text-center">{{_('create_student_account_expl
                         {% endfor %}
                     </select>
                 </div>
+                <!-- START OF NEW QUESTION -->
+                {% if is_teacher %}
+                <div class="signup-container">
+                    <label class="signup-label" for="heard_about">{{_('heard_about_hedy')}}</label>
+                    <ul class="signup-input appearance-none">
+                        <li class="flex" style="justify-content: end;">
+                            <div class="flex flex-row items-center gap-2">
+                                <input type="checkbox" name="heard_about" value="from_another_teacher">
+                                <label style="width: 201px;">{{_('from_another_teacher')}}</label>
```

Comment:
Same here, please style using Tailwind. It might take few tries, but something like `class="w-64"` should work. I personally would like to not style the label at all as it should automatically take the width of it's content. As there is already a `gap-2` on the container `div` this should all work fine.

Reply:
Applied changes! (removed all, same reason as comment above)